packages/ia-components/sandbox/details/ModalGo.jsx

```diff
@@ -18,7 +18,8 @@ class _ModalGo extends React.Component {
   onClick(ev) {
     // ev.currentTarget is the HTML Element on which the onClick sits
     // noinspection JSUnresolvedFunction
-    return AJS.modal_go(ev.currentTarget, this.props.opts);
+    AJS.modal_go(ev.currentTarget, this.props.opts); // Unclear if return from this is what we want (false to run ev.preventDefault)
+    ev.preventDefault(); // Dont propogate event
```

Comment:
pinging again to ask the question - do we need to stop event propagation?

Reply:
Yes - there's a href in that anchor  that is used for non-dweb cases and would otherwise be triggered for dweb